### PR TITLE
Add release notes for version 5.1.1

### PR DESCRIPTION
**Pull request by elfenheart**

### CODE REVIEW

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 1]
```diff
@@ -30,6 +30,36 @@ Cask Data Application Platform Release Notes
...
 
+`Release 5.1.1 <http://docs.cask.co/cdap/5.1.1/index.html>`__
+=============================================================
+
+Improvements
+------------
+
+- :cask-issue:`CDAP-14490` - Google Cloud Spanner sink will create database and table if they do not exist.
+
+- :cask-issue:`CDAP-14542` - Added a Dataset Project config property to the Google BigQuery source to allow reading from a dataset in another project.
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/release-notes.rst:41]: break at 120 chars?

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 2]
```diff
@@ -30,6 +30,36 @@ Cask Data Application Platform Release Notes
...
+
+
+Bug Fixes
+---------
+
+- :cask-issue:`CDAP-10250` - Fixed configuration isolation in mapreduce.
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/release-notes.rst]: hmmm perhaps more details about this?

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 3]
```diff
@@ -30,6 +30,36 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-12229` - Fixed an issue that caused avro, parquet, and orc classes across file, Google Cloud Storage, and S3 plugins to clash and cause pipeline failures.
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/release-notes.rst:47]: break at 120 chars?

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 4]
```diff
@@ -30,6 +30,36 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-14511` - Fixed a bug where plugins that register other plugins would not use the correct id when using the PluginSelector API.
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/release-notes.rst:49]: ditto

File: cdap-docs/reference-manual/source/release-notes.rst (hunk 2 above)
Comment (bdmogal) [reply cdap-docs/reference-manual/source/release-notes.rst]: actually remove this. Its the same as CDAP-14524 below

**Review by bdmogal — APPROVED** [review]
Minor nits. Fix and merge.

File: cdap-docs/reference-manual/source/release-notes.rst (hunk 4 above)
Comment (elfenheart) [reply cdap-docs/reference-manual/source/release-notes.rst:49]: I don't think we do line break for the previous version release notes. Can you confirm we want to do that? @bdmogal

**Review by bdmogal — APPROVED** [review]

**Pull request merged**